pkg/kf/apps/kfapp.go

```diff
@@ -45,28 +46,35 @@ func (k *KfApp) GetNamespace() string {
 	return k.Namespace
 }
 
-func (k *KfApp) getOrCreateRunLatest() *serving.RunLatestType {
-	if k.Spec.RunLatest == nil {
-		k.Spec.RunLatest = &serving.RunLatestType{}
+func (k *KfApp) getOrCreateRunLatest() *serving.RevisionTemplateSpec {
+	if k.Spec.Template == nil {
+		k.Spec.Template = &serving.RevisionTemplateSpec{}
```

Comment:
Does this have the same effect as `RunLatest`, in that this revision will immediately start being served?

Reply:
It does, the traffic will automatically be set to the latest revision unless you manually update the traffic target.